feat: feature gate DefaultPodTopologySpread is GA (#32003)
kubernetes/website: content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```

{{< note >}}
[`SelectorSpread` plugin](/docs/reference/scheduling/config/#scheduling-plugins)
is disabled by default. It's recommended to use `PodTopologySpread` to achieve similar
behavior.
{{< /note >}}

> #### Build-in default constraints

Reviewer: nit: Built-in?

Author: Aha..., good cache. Thanks.

> #### Built-in default constraints

Reviewer 2: We can preserve existing bookmarks etc
```suggestion
#### Built-in default constraints {#internal-default-constraints}
```

Author: updated.

> #### Internal default constraints

Reviewer 2: ```suggestion
### Built-in default constraints {#internal-default-constraints}
```
or
```suggestion
### Automatic default constraints {#internal-default-constraints}
```

Reviewer 3: Built-in sgtm

{{< feature-state for_k8s_version="v1.24" state="stable" >}}

> Kube-scheduler uses the following default topology constraints for the
> `PodTopologySpread` plugin configuration:

Reviewer 2: ```suggestion
If you don't configure any cluster-level default constraints for pod topology spreading,
then then kube-scheduler acts as you specified the the following default topology constraints:
```

Reviewer 3: `as if you`? I'm not a native speaker, so feel free to disregard if I made a mistake :)

Reviewer 2: “as if you” is correct; sorry!

Author: thanks. updated.
